dbt/models/reporting/reporting.ratio_stats.py

```diff
@@ -1,234 +1,238 @@
 # pylint: skip-file
```

Comment:
The diff for this file is large, but the actual changes are very minor; mostly, we undo the indentation we added in https://github.com/ccao-data/data-architecture/pull/453 by pulling the imports and function definitions back out of the `main()` file. Ignoring whitespace should help with reviewing this file again.